homeassistant/helpers/service.py

```diff
@@ -804,8 +812,10 @@ async def entity_service_call(  # noqa: C901
         ]
     )
     assert not pending
-    for future in done:
-        future.result()  # pop exception if have
+
+    response_data: dict[str, ServiceResponse] = {}
+    for entity, task in zip(entities, done):
```

Comment:
Why should we support multiple entities here for the response when we don't do it in general. Can't we raise already here for incorrect call?

Reply:
Fixed.